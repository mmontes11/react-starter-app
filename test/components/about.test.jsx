import React from "react";
import { shallow } from "enzyme";
import About from "components/about";

describe("components/about", () => {
  it("renders an about", () => {
    const wrapper = shallow(<About />);

    expect(wrapper).toMatchSnapshot();
  });
});
