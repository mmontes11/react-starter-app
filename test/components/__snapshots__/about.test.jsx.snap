// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/about renders an about 1`] = `
<div
  className="about"
>
  <p>
    Built by
     
    <a
      href="https://mmontes11.github.io/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Mart√≠n Montes
    </a>
    .
  </p>
  <p>
    More javascript projects available on
     
    <a
      href="https://github.com/mmontes11"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub
    </a>
    .
  </p>
</div>
`;
